src/components/SignUp/SignUp.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+import Button from '../Button/Button'
+import './SignUp.css'
+
+
+const SignUp = () => {
```

Comment:
Add a handleClick here! We don't know what the functionality will be yet, but we want to set it up. Let me know if you have questions about that!